api/integrations/rudderstack/rudderstack.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import typing
+
+import rudder_analytics
+
+from integrations.common.wrapper import AbstractBaseIdentityIntegrationWrapper
+
+if typing.TYPE_CHECKING:
+    from environments.identities.models import Identity
+    from features.models import FeatureState
+
+logger = logging.getLogger(__name__)
+
+
+class RudderstackWrapper(AbstractBaseIdentityIntegrationWrapper):
+    def __init__(self, base_url: str, api_key: str):
+        rudder_analytics.write_key = api_key
+        rudder_analytics.data_plane_url = base_url
+
+    def _identify_user(self, user_data: dict) -> None:
+        rudder_analytics.identify(**user_data)
+        logger.debug(f"Sent event to Rudderstack.")
```

Comment:
Is this really useful at all? Surely we at least want some information about what event was sent and for which identity otherwise it just seems pointless?

Reply:
That's how the analytics integrations work - we send the flag states for an identity into the analytics platform and then its used downstream to create cohorts etc. Other than getting a `200` back from their API there's no information they can send back.

Reply:
Removed